README.md

```diff
@@ -36,7 +36,7 @@ ImpressCMS uses a database to store the data required for running your ImpressCM
 
 #### :rainbow: Theme-based skinnable interface
 
-ImpressCMS is driven by a powerful theme system. Both admins and users can change the look of the entire web site with just a click of a mouse. There are also hundreds of quality themes available for download!
+ImpressCMS is driven by a powerfull theme system. Both admins and users can change the look of the entire web site with just a click of a mouse. There are also hundreds of quality themes available for download!
```

Comment:
'powerful' is the proper English spelling

Reply:
Looks like I need a refresher course on basic English spelling. Wouldn't have made that mistake a few years ago :-(